src/version.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
 module.exports = {
-  defaultVersion: '1.16',
-  supportedVersions: ['1.7', '1.8', '1.9', '1.10', '1.11.2', '1.12.2', '1.13.2', '1.14.4', '1.15.2', '1.16']
+  defaultVersion: '1.16.1',
+  supportedVersions: ['1.7', '1.8', '1.9', '1.10', '1.11.2', '1.12.2', '1.13.2', '1.14.4', '1.15.2', '1.16', '1.16.1']
```

Comment:
Major version duplication (1.16 and 1.16.1) - you just needed to replace `'1.16'` with `'1.16.1'`

Reply:
Fixed